app/views/play/modal/ItemDetailsView.coffee

```diff
@@ -27,6 +27,9 @@ module.exports = class ItemDetailsView extends CocoView
       @componentConfigs = (c.config for c in @item.get('components') when c.config)
 
       stats = @item.getFrontFacingStats()
+      if stats.tooltipDisplay?
+        @$el.find('.stat-label').addClass('has-tooltip').tooltip()
```

Comment:
This doesn't seem to be working? In my testing, I just see default browser tooltips instead of Bootstrap tooltips.